src/tests/ftest/util/job_manager_utils.py

```diff
@@ -964,3 +970,21 @@ def check_logs(self, pattern, since, until, quantity=1, timeout=60):
                 self._command, msg, runtime)
 
         return complete
+
+    def dump_logs(self, hosts=None):
+        """Display the journalctl log data since detecting server start.
+
+        Args:
+            hosts (list, optional): list of hosts from which to display the
+                journalctl log data. Defaults to None which will log the
+                journalctl log data from all of the hosts.
+        """
+        timestamp = None
+        if self.timestamps["running"]:
+            timestamp = self.timestamps["running"]
+        elif self.timestamps["verified"]:
+            timestamp = self.timestamps["verified"]
+        if timestamp:
+            if hosts is None:
+                hosts = self._hosts
```

Comment:
Can you not just make `self._hosts` the default value for `hosts` in the method signature?  I don't recall ever trying to use a `self` member as a default value myself though.  Not sure if it works or not.  But that seems to be what you want here.

Reply:
No as `self` itself is one of the method arguments.

Reply:
Of course.